admin/class-admin-apple-themes.php

```diff
@@ -746,9 +759,15 @@ private function _save_edit_theme() {
 	 */
 	private function _set_theme() {
 
+		// Check the nonce.
```

Comment:
You seem to use this same pattern a lot of checking a nonce, maybe you could make this a separate function in the future.

Reply:
I'd love to, but `phpcs` yells at you if you try to abstract the nonce check. It wants the nonce check at the top of every function that touches form data.